src/Classes/Options/Particles/Shape/CharacterShape.ts

```diff
@@ -0,0 +1,17 @@
+import {ICharacterShape} from "../../../../Interfaces/Options/Shape/ICharacterShape";
+
+export class CharacterShape implements ICharacterShape {
+    public fill: boolean;
+    public font: string;
+    public style: string;
+    public value: string | string[];
+    public weight: string;
+
+    constructor() {
+        this.fill = false;
+        this.font = "Verdana";
+        this.style = "";
+        this.value = "*";
+        this.weight = "400";
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)